go.mod

```diff
@@ -456,7 +456,7 @@ replace (
 	github.com/openshift/cluster-api-provider-libvirt => github.com/openshift/cluster-api-provider-libvirt v0.2.1-0.20191219173431-2336783d4603
 	github.com/openshift/cluster-api-provider-ovirt => github.com/openshift/cluster-api-provider-ovirt v0.1.1-0.20211215231458-35ce9aafee1f
 	github.com/openshift/console-operator => github.com/openshift/console-operator v0.0.0-20220318130441-e44516b9c315
-	github.com/openshift/installer => github.com/jewzaam/installer-aro v0.9.0-master.0.20220524230743-7e2aa7a0cc1a
+	github.com/openshift/installer => github.com/ventifus/installer-aro v0.0.0-20230404230942-a99152bf4d63
```

Comment:
should this get reverted? or are you the new owner of `installer-aro`?

Reply:
The intent here was to make CI and such pass. This won't be needed and I'll revert once those changes are merged into installer-aro.